.eslintrc.js

```diff
@@ -51,64 +50,37 @@ module.exports = {
         ],
       },
     ],
-    'nonblock-statement-body-position': 'error',
-    // Airbnb restricts isNaN and isFinite which are necessary for IE 11
-    // we have to be disciplined about the usage and ensure the Number type for its
-    // arguments
-    'no-restricted-globals': ['error'].concat(confusingBrowserGlobals),
```

Comment:
Do we allow confusing browser globals now or is this replaced by another rule?

Reply:
This new rule is stricter, basically *error* without any exceptions like we used to have. You have seen it with the isNaN -> Number.isNaN change I have tried in the test you asked to revert.